src/sentry/api/serializers/models/group.py

```diff
@@ -35,6 +35,9 @@
 
 @register(Group)
 class GroupSerializer(Serializer):
+    def __init__(self, environment_id_func=None):
+        self.environment_id_func = environment_id_func if environment_id_func is not None else lambda: None
```

Comment:
This ternary (and default) is temporary, @bretthoerner is going to make this required so don't get too caught up in stylistic concerns. I mainly wrote it as a ternary to not have to write it as a `def` statement because autoformatter

Reply:
This also doesn't need to be on `GroupSerializer` yet (it's only used in `StreamGroupSerializer` when the `stats_period` is not none) but @bretthoerner needs it at this level and would have ended up moving it anyway so I put it here to reduce churn

Reply:
> This ternary (and default) is temporary

This wasn't temporary at all